src/Artsy/Router/Route.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * Taken from @taion as per example of how they actually use found-relay and
+ * have default setup for each route.
+ *
+ * TODO:
+ * - Decide if we want to change our getDataFrom, such as the one @taion uses.
+ * - There's opportunity to unify some code with renderWithLoadProgress.
+ */
+
+import { Spinner } from "@artsy/palette"
+import { LoadingClassName } from "Artsy/Relay/renderWithLoadProgress"
+import BaseRoute from "found/lib/Route"
+import React from "react"
+import styled from "styled-components"
+
+const SpinnerContainer = styled.figure`
+  width: 100%;
+  height: 100px;
+  position: relative;
```

Comment:
`import { Spinner } from '@artsy/palette'`

Reply:
That'a above. This is just a container.